urumanifest/github.py

```diff
@@ -374,6 +375,12 @@ def nuke_recurse(nuke: Path):
     database.current_sha = rev
     database.valid = True
 
+class ArtifactInfo(NamedTuple):
+    path: Path
+    zipinfo: zipfile.ZipInfo
+    name: str
+    bundle: bool = False
+
```

Comment:
```suggestion
class ArtifactInfo(NamedTuple):
    path: Path
    zipinfo: zipfile.ZipInfo
    name: str
    bundle: bool = False


```

PEP8 wants two blank lines after classes.